src/python/WMCore/MicroService/Unified/MSTransferor.py

```diff
@@ -1,102 +1,83 @@
-#!/usr/bin/env python
 """
-_MSTransferor_
-
-Class to hold the whole logic behind the transferor module
+File       : MSTransferor.py
+Author     : Valentin Kuznetsov <vkuznet AT gmail dot com>
```

Comment:
How come you put your name on the module I created? :)

Reply:
hmm, I thought I created MS codebase, you just mess it up :) But since you asked I put your name and will blame you for not working code from now on.